### PR TITLE
Accommodate changes to key interface refactoring

### PR DESCRIPTION
**Pull request by rpdome**
ref: https://github.com/AzureAD/ad-accounts-for-android/pull/1700

### CODE REVIEW

File: common4j/src/main/com/microsoft/identity/common/java/crypto/IDecryptor.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    /**
+     * Decrypt the given byte array.
+     *
+     * @param key               the key to decrypt with.
+     * @param decryptAlgorithm  algorithm to decrypt with.
+     * @param iv                an initialization vector (IV).
+     * @param dataToBeDecrypted the data to be encrypted.
+     * @return a decrypted byte array.
+     */
+    byte[] decrypt(@NonNull final Key key,
```
Comment (shahzaibj) [inline common4j/src/main/com/microsoft/identity/common/java/crypto/IDecryptor.java:48]: I think this is the 3rd place we have a decrypt method on an interface

File: common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java [hunk 2]
```diff
@@ -231,7 +231,7 @@ public static String getStackTraceAsString(@NonNull final Exception exception) {
         return pw.toString();
     }
 
-    public static String fromByteArray(@NonNull final byte[] bytes) {
```
Comment (shahzaibj) [inline common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java:234]: So Nullable?

Comment (shahzaibj) [reply common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java:234]: what happens if this is NULL?

**Review by shahzaibj — APPROVED** [review]
Left minor comments, but LGTM

File: common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java (hunk 2 above)
Comment (rpdome) [reply common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java:234]: a warning is shown if we put NonNull here, so I removed it.

**Review by AdamBJohnsonx — APPROVED** [review]

**Pull request merged**